src/redux/Categories/categories.js

```diff
@@ -0,0 +1 @@
+// Redux
```

Comment:
- [ ] Please make sure to have actions and reducers for  Categories as you have done for Books. It's a requirement for this project.

   You are doing great in this project :+1: 
   
   Wish you all the best :pray: